addons/sale_product_configurator/static/src/js/product/product.xml

```diff
@@ -1,70 +1,31 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<templates xml:space="preserve">
-    <t t-name="sale_product_configurator.product">
-        <td class="o_sale_product_configurator_img">
-            <img
-                t-if="this.props.id"
-                t-att-src="'/web/image/product.product/'+this.props.id+'/image_128'"
-                alt="Product Image"/>
-            <img
-                t-else=""
-                t-att-src="'/web/image/product.template/'+this.props.product_tmpl_id+'/image_128'"
-                alt="Product Image"/>
-        </td>
-        <td t-att-colspan="this.props.optional ? 2:false">
-            <div class="o_sale_product_configurator_name">
-                <strong t-out="this.props.display_name"/>
-                <div
-                    t-if="this.props.description_sale"
-                    t-out="this.props.description_sale"
-                    class="text-muted small"/>
-            </div>
-            <t t-foreach="this.props.attribute_lines" t-as="ptal" t-key="ptal.id">
-                <PTAL t-props="ptal" productTmplId="this.props.product_tmpl_id"/>
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="saleProductConfigurator.product" t-inherit="product.product_configurator.product" t-inherit-mode="extension">
+        <xpath expr="//td[last()]" position="after">
+            <t t-if="this.env.options.showPrice">
```

Comment:
It does because of the extension of the ``product.product_configurator.product``
As it will be used in both modules, we (at least for now) do not need to display the price in the ``product_configurator``. So we have to specify in sales that we want to display the price.
To remove it from here, we would need to change the ``t-inherit-mode`` to primary and redefine a new component to use that new template. The problem here is that we would be forced to change other templates to use those new components. There is no best solution in this case, either we rewrite a lot of code, either we add a parameter to limit the code duplication.